dubbo-config/pom.xml

```diff
@@ -49,4 +48,16 @@
             <scope>test</scope>
         </dependency>
     </dependencies>
+
+    <profiles>
+        <profile>
+            <id>jdk-version-ge-17</id>
+            <activation>
```

Comment:
Add release profile to ensure that `dubbo-config-spring6` is compiled when releasing

Reply:
Done.